frontend/internal-packages/agent/tsconfig.json

```diff
@@ -3,8 +3,7 @@
   "compilerOptions": {
     "outDir": "dist",
     "declaration": true,
-    "target": "ES2022",
-    "moduleResolution": "node"
```

Comment:
Revert to the default `bundler`.

https://github.com/liam-hq/liam/blob/d246aab7fccfd3bfed8cbb4c81d9dfcac4340a2e/frontend/internal-packages/configs/tsconfig/base.json#L11